### DISCUSSION TITLE
I don't want updates on repositories i starred

### DISCUSSION THREAD

**Discussion by mishushakov** [community/community#13318] [Other Feature Feedback, Questions, & Ideas]
For example:

<img width="881" alt="Screenshot 2022-03-23 at 10 29 51" src="https://user-images.githubusercontent.com/10400064/159667428-a13abc67-acd5-4e5c-bcae-fbb1642fbb23.png">

I have starred the repositories in the past, but i'm not interested in release updates!

I have starred 800+ repositories, should i now _unstar_ them so the GitHub feed doesn't keep spamming me about updates? I'm not watching any repositories suggested nor am i following the orgs/authors

my suggestion is the release updates should only appear for repositories i am watching and from authors i'm following

**Comment by sntwandy** [comment]
It should have an option to receive yes or no updates in the feed, that would be good for those of us who have many starred repos.

**Comment by msharov** [comment]
It would be better for you to not star repositories you do not care about in the first place. While I do appreciate that more stars make my projects more visible, I do not want "pity stars" just for this purpose, because that's just lying both to me and other users of my project. If you star my project, it should be because you care about it, not because having more stars is a status symbol.

**Reply by kabirbaidhya** [reply]
> If you star my project, it should be because you care about it

Agreed. But caring about a project does not mean you want to be spammed with everything that is going on with all the projects that you've starred. IMO, feed subscription should be opt-in. Otherwise it is just a noise that helps no one. 

Eg: you might have starred over 1000s of projects you've used over time, but you don't always use same tools and you most likely would want to stay up-to-date with only a handful of them based on your current toolset.

**Reply by lo48576** [reply]
> If you star my project, it should be because you care about it

I star projects because I'm actually interested in the projects, but...

* Independent "Watch" feature already exists for watching repository activity, so why the star should have duplicated meaning?
* I'm not necessarily interested in the repository RIGHT NOW, even if I'll definitely revisit LATER when I'm free.
* For some (or actually many) packages I receive updates notification from different channels, such as from package managers or other package repository (for each programming languages) for example.

Sending lots of notifications about projects I don't "Watch" is nothing but spamming.
If I need them, I'll "Watch". "Star" should have nothing to do with this.

**Reply by mpaperno** [reply]
Totally ridiculous suggestion, sorry... that was the whole point of "starring" a repo vs. "watching" it.  Now there's 100s of feeds all of a sudden because I starred a repo in the last 10 years (or whatever).  That's called a "regression issue" if I've ever seen one.

"Pity stars?"  Honestly, no offense but it doesn't matter how _you_ want people to use stars, what matters is how others actually have used them since they were "a thing."

**Comment by mishushakov** [comment]
ok, what if i starred a repo in the past but don't care about it anymore? Should i _unstar_ it?

star for me is equivalent to like, i use it to show my appreciation to a project
but if i'm not watching a project, why should i receive releases updates?

**Comment by msharov** [comment]
Yes, you should unstar it. If you don't care about my project any more, that is useful information both to me, so I know when people stop using my project, and to other people, so they are not misled into thinking my project is more popular than it really is. The same logic should apply to "Likes". If you don't like it any more, should you not say so?

**Reply by veloxy** [reply]
People have been using stars as bookmarks for a long time, and then suddenly all the release info was shown on dashboard.

The new lists addition encourages the behavior to star to bookmark a repository. I'm not interested in every release of everything I star.

**Comment by kabirbaidhya** [comment]
Showing up news from all the repositories I've starred is just too much noise. I don't find any of them useful. 
Suggestion: Show feed from the ones that I **watch**. And, I can un-watch the ones which I do not want to receive feed anymore.
